ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1094 @@
+
+[#_running_on_openshift_start_postgresql_database]
+===== Running on OpenShift: Start PostgreSQL database
+
+For the test purposes you may use
+
+[source,sh,options="nowrap",subs="+quotes,attributes+"]
+----
+# change path to ${PATH_TO_QUICKSTART_DIR}
+cd ${PATH_TO_QUICKSTART_DIR}
+# deploy not-production ready PostgreSQL database to OpenShift
+oc create -f ejb-txn-remote-call/client/extensions/postgresql.deployment.yaml
+----
+
+[#_running_on_openshift_build_the_application]
+==== Running on OpenShift: Build the application
+
+OpenShift provides the s2i functionality out-of-the-box.
+The quickstart defines two OpenShift templates which setup `ImageStream`s
+and the `BuildConfig`.The `BuildConfig` builds the application based on the images
+configured in thee `ImageStream`.We named the `BuildConfig` template `wildfly-s2i`.
+
+[source,sh,options="nowrap",subs="+quotes,attributes+"]
+----
+cd ${PATH_TO_QUICKSTART_DIR}
+
+# define the image streams of app server images
+oc create -f ejb-txn-remote-call/openshift-image-streams.json
+# upload template definition for the builds named wildfly-s2i
+oc create -f ejb-txn-remote-call/openshift-chain-build-config.json
+
+# invoke the build via s2i chain build
+oc new-app --template=wildfly-s2i -p IMAGE_STREAM_NAMESPACE=$(oc project -q) \
+  -p APPLICATION_NAME=client -p SOURCE_REPOSITORY_URL=https://github.com/wildfly/quickstart.git \
+  -p SOURCE_REPOSITORY_REF=master -p CONTEXT_DIR=ejb-txn-remote-call/client
+oc new-app --template=wildfly-s2i -p IMAGE_STREAM_NAMESPACE=$(oc project -q) \
+  -p APPLICATION_NAME=server -p SOURCE_REPOSITORY_URL=https://github.com/wildfly/quickstart.git \
+  -p SOURCE_REPOSITORY_REF=master -p CONTEXT_DIR=ejb-txn-remote-call/server
+----
+
+With this we have created the `ImageStream` which contains the docker image
+of the {productName} runtime image with the deployed applications of this quickstart.
+
+Now you need to wait until the builds are finished.It will probably be a few minutes.
+You can verify that the build is done by running `oc get pod`.
+Then all the `*-build` jobs are with `STATUS` `Completed`.
+
+[source,sh,options="nowrap"]
+----
+oc get pod
+NAME                                READY   STATUS      RESTARTS   AGE
+client-2-build                      0/1     Completed   0          35m
+client-build-artifacts-1-build      0/1     Completed   0          45m
+server-2-build                      0/1     Completed   0          15m
+server-build-artifacts-1-build      0/1     Completed   0          19m
+----
+
+[#_running_on_openshift_install_productname_operator]
+==== Running on OpenShift: Install {productName} Operator
+
+For installing the {productName} Operator to OpenShift there are a few steps required.
+The best they are summarized at
+https://github.com/wildfly/wildfly-operator/blob/master/build/run-openshift.sh[build/run-openshift.sh].
+The script setup the OpenShift and creates the CRDs necessary for {productName} Operator
```

Comment:
"The script setup the OpenShift" -> "The script sets up OpenShift"

Reply:
+1